newton/tests/test_contact_sensor.py

```
@@ -0,0 +1,237 @@
+                assert_np_equal(net_forces[1], scenario["force_B_vs_A"])
+                assert_np_equal(net_forces[2], scenario["force_A_vs_All"])
+                assert_np_equal(net_forces[3], scenario["force_B_vs_All"])
+
+
+class TestContactSensorManager(unittest.TestCase):
+    def test_add_sensor(self):
+        device = wp.get_device()
+        model = MockModel(device)
+        cm = ContactSensorManager(model)
+
+        contact_info = create_contact_info(
```

Ruff (0.12.2)

Local variable `contact_info` is assigned to but never used

Remove assignment to unused variable `contact_info`

(F841)